app/views/user_profiles/show.html.haml

```diff
@@ -1,3 +1,5 @@
+- content_for :before_title, "#{@user.username} — "
```

Comment:
> This change should affect the user profile 'show' page, which you can reach by clicking your own username at the upper right while logged in.

in my application i don't see any changes

Reply:
Can you post a screenshot of that view?

Reply:
yes, please do.

Reply:
The other pages i mean it's dashboard,Training,findprogram,Report a problem and documentations right?

Reply:
Some of those link to places outside of the app, and some may already have a useful title. You can just explore around the app to find places that don't have a useful title.

Reply:
cool

Reply:
![page2](https://user-images.githubusercontent.com/22706319/137212774-2ecc968e-7af7-4400-9191-8812e97ecdb5.png)
Found this page add  added a title to it

Reply:
![surveyResults](https://user-images.githubusercontent.com/22706319/137220961-7b767f38-89c8-4754-8e58-848ff39628c4.png)
add the word 'Survey'' to Results because the url was pointing to 'Survey/results'. The other pages seems to have meaningful titles

Reply:
The titles are what appears in the Tab (and in the history if you click and hold the browser 'back' arrow). The survey views (Question Groups, Surveys, Assignment, Results, Recent Responses) all just have the application name (Programs & Events Dashboard, for your instance) as the title.

Reply:
![url](https://user-images.githubusercontent.com/22706319/137293121-c3367c42-03bb-4049-9948-27f8fe87ec1f.jpeg)
i think this is the implementation your are taking about right?

Reply:
Sure.